demos/apps/bookinfo/README.md

```diff
@@ -343,10 +341,12 @@ If you now refresh the `productpage` you'll see that while the load generator is
    $ ./cleanup.sh
    ```
 
+1. Optionally shut down the control plane services using the cleanup instructions [here](../../../kubernetes/README.md).
```

Comment:
100 times???

"Note: With the Envoy sidecar implementation, you may need to refresh the productpage 100 times to see the proper distribution."